kiwi/constraint.h

```diff
@@ -28,15 +28,15 @@ class Constraint
 {
 
 public:
-    Constraint() : m_data(0) {}
+    Constraint() = default;
```

Comment:
Are we sure that `m_data(0)` is the equivalent of the default initialization of `m_data`?

Reply:
Yes it is equivalent, but actually this is not required as this is what the default constructor will do, I'll leave it `= default;`